### DISCUSSION TITLE
Add legacy beatmap countdown from stable

### DISCUSSION THREAD

**Discussion by nagi-desuuu** [ppy/osu#20583] [Ideas] [answered]
Due to a severe lack for appropriate words, I'll let this video speak for itself.

https://user-images.githubusercontent.com/75596775/193992679-affa702b-4cdb-4d02-9f0f-2b8cec080702.mp4

(Side note, some beatmaps actually included a countdown, such as Disco Prince)

**Comment by peppy** [comment, marked as answer]
#20584
